backend/server/src/db/migrations/4-series.sql

```diff
@@ -0,0 +1,20 @@
+call prepare_randomized_ids('series');
+
+create table series (
+    id bigint primary key default randomized_id('series'),
+
+    -- Opencast internal data
+    opencast_id text not null,
+    created_by text not null,
+
+    -- Meta data
+    title text not null,
+    subject text,
+    description text,
+    language text,
+    rights text,
```

Comment:
Is `rights` actually used?

Reply:
:shrug:

Reply:
@lkiesow can probably answer that better than me.

Reply:
I have seen it used but very rarely. We could push that to the additional metadata if we want to.

Reply:
I tend to put more stuff in an "additional metadata" column, I guess. Unless we have a strong suspicion that we will want to search through that metadata field, or filter or sort by it, I would not add it for now. But that's just me.

Reply:
I have no hard opinion. I also don't have a strong suspicion that we will want to search/filter/order using this column, so maybe it should just go. :thinking: